packages/bibtex/csl/engine.lua

```diff
@@ -449,7 +463,10 @@ function CslEngine:_layout (options, content, entries)
    -- quite logically as we force a paragraph break between entries.
    for _, entry in ipairs(entries) do
       self:_prerender()
-      local elem = self:_render_children(content, entry)
+      local elem = self:_render_children(content, entry, {
+         secondFieldAlign = self.inheritable.bibliography["second-field-align"] and true or false,
+         delimiter = options.delimiter
```

Comment:
Useless: There shouldn't be a delimiter on the bibliography layout as per the specs.

Reply:
Fixed